src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * firstLeg * secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: triangle, area: "
+                + df.format(getArea()) + " sq. units, firstLeg: "
+                + df.format(firstLeg) + " units, secondLeg: "
+                + df.format(secondLeg) + " units, color: "
+                + getColor());
```

Comment:
Directly using `System.out.println` for drawing the figure is not ideal for larger applications as it couples your drawing logic with the console output. Consider using a dedicated method to return the figure's description as a String and then print it elsewhere in your application.